### PR TITLE
Nuxt initial setup

### LINKED ISSUE #4: Initial Nuxt app setup

### PR DESCRIPTION
**Pull request by hoo761**
Created initial nuxt project with the sample creator tool [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).

Closes #4

### CODE REVIEW

File: scrud-nuxt/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```
Comment (dcharbon) [inline scrud-nuxt/.editorconfig:1]: Should this file be committed?

**Review by dcharbon — COMMENTED** [review]
Should `.editorconfig` be committed? Is this a personal dotfile?

File: scrud-nuxt/.editorconfig (hunk 1 above)
Comment (hoo761) [reply scrud-nuxt/.editorconfig:1]: Yes, production openteams-nuxt uses it and it is super useful while developing. It is supported by most editors and helps with styling the code. [editorconfig.org](https://www.editorconfig.org).

Comment (hoo761) [reply scrud-nuxt/.editorconfig:1]: @dcharbon

**Review by dcharbon — APPROVED** [review]

**Pull request merged**